### PR TITLE
- Update gui from iml-gui@6.0.13 to iml-gui@6.1.0

### PR DESCRIPTION
**Pull request by johnsonw**
- Added repo_name var in spec file
- Updated items in spec file to use %{name} where possible and
  %{repo_name} where appropriate

Signed-off-by: Will Johnson <william.c.johnson@intel.com>

### CODE REVIEW

File: iml-gui/iml-gui.spec [hunk 1]
```diff
@@ -1,11 +1,12 @@
 Name:       iml-gui
```
Comment (jgrund) [inline iml-gui/iml-gui.spec]: Do we want to use repo_name here?

File: iml-gui/iml-gui.spec [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+Version:    6.1.0
 Release:    1%{?dist}
 Summary:    Graphical User Interface for Intel Manager for Lustre.
 License:    MIT
 Group:      System Environment/Libraries
-URL:        https://github.com/intel-hpdd/gui
-Source0:    http://registry.npmjs.org/@iml/%{name}/-/%{name}-%{version}.tgz
+%define repo_name gui
+URL:        https://github.com/intel-hpdd/%{repo_name}
+Source0:    http://registry.npmjs.org/@iml/iml-%{repo_name}/-/%{repo_name}-%{version}.tgz
```
Comment (jgrund) [inline iml-gui/iml-gui.spec]: no need for iml- here

Comment (johnsonw) [inline iml-gui/iml-gui.spec]: good catch, removed

**Comment by johnsonw** [comment]
repushed

**Pull request merged**